packages/core/util/tracks.js

```diff
@@ -72,11 +72,15 @@ export function guessAdapter(fileName, protocol) {
 
   if (/\.cram$/i.test(fileName))
     return {
-      type: UNSUPPORTED,
+      type: 'CramAdapter',
+      cramLocation: { [protocol]: fileName },
+      craiLocation: { [protocol]: `${fileName}.crai` },
     }
   if (/\.crai$/i.test(fileName))
     return {
-      type: UNSUPPORTED,
+      type: 'CramAdapter',
+      cramLocation: { [protocol]: fileName.replace(/\.csi$/i, '') },
```

Comment:
Why is csi coming up here

Reply:
Maybe crai instead :)

Reply:
Yep, should've been crai. Thanks for the catch. Fix'd.